html_app/ui/MainFrame.js

```diff
@@ -282,270 +291,263 @@ scb.ui.MainFrame = function scb_ui_MainFrame(master_model, context) {
-  					}
-					catch(err)
-  					{
-  							var popoutWindow =window.open("ug2/full_guide.html#"+popout_string);
-  					}
-					
-			});
-			
-			$("#closesearch").click(function(){
-				$('.help_search_input').val('');
-    			$("#closesearch").hide();
-				$(".scb_s_ug_home").click();
-			});
-
-		});
-		
+
     });
 
-	//HANDLER FOR LOGIN BUTTON AND IFRAME
+    //HANDLER FOR LOGIN BUTTON AND IFRAME
     scb.utils.off_on(workarea.parent(), 'click', '.scb_f_login', function (evt) {
         scb.ui.static.MainFrame.ensure_auth_context();
-        if(assignments.selected && !user_is_auth){
-				$('html').css('overflow', 'hidden');
-				$('body').prepend(scb_experiment_setup.general_error_overlay());
-
-
-				$.jqDialog.confirm("If you sign in to your account, you will lose your current work as a guest. Would you like to continue?",
-					function() {
-						$('html').css('overflow', 'visible');
-						$('.error_overlay').remove();
-						if (user_is_auth) {
-			
-							window.location = '/accounts/logout/';
-						}
-						else {
-							add_login_script(workarea);
-						}
-						evt.preventDefault();
-					},// callback function for 'YES' button
-					function() {
-							$('.error_overlay').remove();
-							$('html').css('overflow', 'visible');
-							return;
-					}		// callback function for 'NO' button
-				);
-				$('.jqDialog_header').remove();
-				$('#jqDialog_box').prepend(scb_experiment_setup.experiment_error());
-				$('#jqDialog_box').attr('role', 'alertdialog');
-				evt.preventDefault();
+        if (assignments.selected && !user_is_auth) {
+            $('html').css('overflow', 'hidden');
+            $('body').prepend(scb_experiment_setup.general_error_overlay());
+
+
+            $.jqDialog.confirm("If you sign in to your account, you will lose your current work as a guest. Would you like to continue?",
+                function () {
+                    $('html').css('overflow', 'visible');
+                    $('.error_overlay').remove();
+                    if (user_is_auth) {
+
+                        window.location = '/accounts/logout/';
+                    }
+                    else {
+                        add_login_script(workarea);
+                    }
+                    evt.preventDefault();
+                },// callback function for 'YES' button
+                function () {
+                    $('.error_overlay').remove();
+                    $('html').css('overflow', 'visible');
+                    return;
+                }		// callback function for 'NO' button
+            );
+            $('.jqDialog_header').remove();
+            $('#jqDialog_box').prepend(scb_experiment_setup.experiment_error());
+            $('#jqDialog_box').attr('role', 'alertdialog');
+            evt.preventDefault();
+        }
+        else {
+            if (user_is_auth) {
+
+                window.location = '/accounts/logout/';
```

Comment:
Can this at least be declared at the top of the file as `LOGIN_URL`, or in some common JS file preferably, so if it changes, we don't have to find it all over the place?